sentry-android-core/src/main/java/io/sentry/android/core/AppLifecycleIntegration.java

```diff
@@ -83,13 +84,25 @@ private void addObserver(final @NotNull IHub hub) {
     if (this.options == null) {
       return;
     }
+
+    Lifecycle lifeCycle;
+
+    try {
+      lifeCycle = ProcessLifecycleOwner.get().getLifecycle();
+    } catch (AbstractMethodError e) {
+      options
+          .getLogger()
+          .log(SentryLevel.ERROR, "AppLifecycleIntegration failed to get Lifecycle.", e);
+      return;
+    }
+
     watcher =
         new LifecycleWatcher(
             hub,
             this.options.getSessionTrackingIntervalMillis(),
             this.options.isEnableAutoSessionTracking(),
             this.options.isEnableAppLifecycleBreadcrumbs());
-    ProcessLifecycleOwner.get().getLifecycle().addObserver(watcher);
+    lifeCycle.addObserver(watcher);
```

Comment:
Actually based on the stack trace, `addObserver` is throwing and not the `ProcessLifecycleOwner.get()`
I'd rather wrap the whole L92 within a try/catch, catching a `Throwable` even and not a specific  `AbstractMethodError`, and add a comment about the `AbstractMethodError` problem.

Reply:
Can I just set the `watcher = null` to avoid removing it during close?

Reply:
It threw where he got the guard unless I'm missing something: addObserver(AppLifecycleIntegration.java:92
Original line 92